Bottom: 6f0b59019ab49bdd0f65e29b9bbe191d6e488c0b
Top:    8b7c5fb27474fda3e349ca01c559179cc4a573b3
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-24 21:03:16 -0400

Refresh of tas5504-codec

---

diff --git a/sound/soc/codecs/tas5504.c b/sound/soc/codecs/tas5504.c
index e3948de..90f299a 100644
--- a/sound/soc/codecs/tas5504.c
+++ b/sound/soc/codecs/tas5504.c
@@ -843,7 +843,7 @@ static const struct i2c_device_id tas5504_device_id[] = {
 	{ "tas5504", 0 },
 	{ }
 };
-MODULE_DEVICE_TABLE(i2c, tas5504_id);
+MODULE_DEVICE_TABLE(i2c, tas5504_device_id);
 
 static struct i2c_driver tas5504_driver = {
 	.driver		= {
